### PR TITLE
PSTN and STN config changes

### PR DESCRIPTION
**Pull request by Daniel-VDM**
Changes for Mkeys

### CODE REVIEW

File: configs/benchmark-ps.json [hunk 1]
```diff
@@ -53,13 +53,13 @@
     "dashboard": false,
     "crosstx": 30,
     "attacked_mode": 0,
-    "peer_per_shard": 10,
-    "minpeer": 6,
+    "peer_per_shard": 15,
+    "minpeer": 8,
```
Comment (LeoHChen) [inline configs/benchmark-ps.json]: use 3 for minpeer, we only have 5 nodes, minpeer is at node level.

File: configs/benchmark-stn.json [hunk 2]
```diff
@@ -1,21 +1,21 @@
...
+  "description": "aws 160 nodes, 4 shards",
   "client": {
     "num_vm": 0,
-    "type": "t3.medium",
+    "type": "m5a.large",
     "regions": "iad"
   },
   "leader": {
-    "num_vm": 1,
+    "num_vm": 4,
```
Comment (LeoHChen) [inline configs/benchmark-stn.json]: num_vm: 1, as we have 4 regions already, num_vm is per region number

File: configs/benchmark-stn.json [hunk 3]
```diff
@@ -1,21 +1,21 @@
...
     "type": "m5a.large",
-    "regions": "iad,pdx,cmh",
+    "regions": "iad,pdx,cmh,sfo",
     "protection": false,
     "root": 15
   },
   "explorer_node": {
-    "num_vm": 1,
+    "num_vm": 4,
```
Comment (LeoHChen) [inline configs/benchmark-stn.json]: 1

File: configs/benchmark-stn.json [hunk 4]
```diff
@@ -48,18 +48,18 @@
...
-    "shards": 3,
+    "shards": 4,
     "duration": 120,
     "dashboard": false,
     "crosstx": 30,
     "attacked_mode": 0,
-    "peer_per_shard": 250,
-    "minpeer": 168,
+    "peer_per_shard": 120,
+    "minpeer": 80,
```
Comment (LeoHChen) [inline configs/benchmark-stn.json]: 30, as we have only 40 nodes

File: configs/benchmark-ps.json (hunk 1 above)
Comment (Daniel-VDM) [reply configs/benchmark-ps.json]: Updated to 3

File: configs/benchmark-stn.json (hunk 2 above)
Comment (Daniel-VDM) [reply configs/benchmark-stn.json]: oh okay, changed to 1

File: configs/benchmark-stn.json (hunk 4 above)
Comment (Daniel-VDM) [reply configs/benchmark-stn.json]: yep, changed to 30

File: configs/benchmark-stn.json (hunk 3 above)
Comment (Daniel-VDM) [reply configs/benchmark-stn.json]: changed to 1

**Pull request merged**